tools/configen/tests/test_modules/future_annotations.py

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+from __future__ import annotations
+
+from dataclasses import dataclass
+from enum import Enum
+from typing import List, Optional
+
+from omegaconf import MISSING
```

Comment:
Can you reduce the scope of this test?
generate a single class that has enough richness inside it.

Reply:
and done!